CONTRIBUTING.md

```diff
@@ -194,6 +194,12 @@ There is a "stats" script that will generate various statistics about layer comp
 - `npm run stats -- -c` - total layer count
 - `npm run stats -- -h` - list all options
 
+There is a "util" script that will extract layer style data:
+
+- `node scripts/util -pl <layer>` - JSON contents of a specified layer
```

Comment:
I deliberately did not add this script to `package.json` because it produces JSON output, which is disrupted by the npm info logging that happens before the JSON is printed.

Reply:
It doesn’t need to be part of `npm run build`, just an extra optional run script like the existing `npm run style`:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/9eb5b36019ced9a49507e137bad3cb50220878eb/package.json#L32

Reply:
Are you envisioning that the optional script would also output to a file?  I was thinking that the only reasonable use would be to pipe the output to "something else" and the info logging breaks that.

Reply:
Resolved by documenting -s option